backend/app/Http/Controllers/User/ShowAllUserController.php

```diff
@@ -20,9 +22,39 @@ class ShowAllUserController extends Controller
      */
     public function __invoke(): View|Factory
     {
-        $users = User::where("id", ">", "1")->get();
-        $projects = Project::where("id", ">", "1")->get();
+        //ユーザー情報入力済みのユーザーを引っ張り出す
+        $users = UserInfo::with('user:id,name,profile_photo_path')->get([
+            'id',
+            'status',
+            'grade',
+            'u_u_major_id',
+        ]);
 
-        return view('user.index', ["users" => $users, "projects" => $projects,]);
+
+        $listedUsers = [];
+        foreach ($users as $user) {
+            //ユーザー情報のu_u_major_idの情報をもとにUUMajorを取得
+            $uuMajor =  UUMajor::find($user->u_u_major_id);
+            if ($uuMajor) {
+                $user->uuMajor = $uuMajor->name;
+                $user->uuFaculty = $uuMajor->faculty_id->label();
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Access to an undefined property App\Models\UUMajor|Illuminate\Database\Eloquent\Collection<int, App\Models\UUMajor>::$faculty_id.